src/kernel/task.zig

```diff
@@ -152,28 +152,33 @@ test "create expected setup" {
     // Will allocate the first PID 1, 0 will always be allocated
     expectEqual(task.pid, 1);
 
-    // This looks pointless tbh as it is just copy paste from init function
-    // but in case something changes
-    expectEqual(task.stack[arch.STACK_SIZE - 18], 0x10);
-    expectEqual(task.stack[arch.STACK_SIZE - 17], 0x10);
-    expectEqual(task.stack[arch.STACK_SIZE - 16], 0x10);
-    expectEqual(task.stack[arch.STACK_SIZE - 15], 0x10);
-    expectEqual(task.stack[arch.STACK_SIZE - 14], 0x10);
-    expectEqual(task.stack[arch.STACK_SIZE - 13], 0);
-    expectEqual(task.stack[arch.STACK_SIZE - 12], 0);
-    expectEqual(task.stack[arch.STACK_SIZE - 11], @ptrToInt(&task.stack[arch.STACK_SIZE - 1]));
-    expectEqual(task.stack[arch.STACK_SIZE - 10], 0);
-    expectEqual(task.stack[arch.STACK_SIZE - 9], 0);
-    expectEqual(task.stack[arch.STACK_SIZE - 8], 0);
-    expectEqual(task.stack[arch.STACK_SIZE - 7], 0);
-    expectEqual(task.stack[arch.STACK_SIZE - 6], 0);
-    expectEqual(task.stack[arch.STACK_SIZE - 5], 0);
-    expectEqual(task.stack[arch.STACK_SIZE - 4], 0);
-    expectEqual(task.stack[arch.STACK_SIZE - 3], @ptrToInt(test_fn1));
-    expectEqual(task.stack[arch.STACK_SIZE - 2], 0x08);
-    expectEqual(task.stack[arch.STACK_SIZE - 1], 0x202);
-
-    expectEqual(task.stack_pointer, @ptrToInt(&task.stack[arch.STACK_SIZE - 18]));
+    switch (builtin.arch) {
```

Comment:
Oh I meant put this in arch.zig, where `initTaskStack` is defined.

Reply:
In that case, it isn't really testing anything as it is all hard coded values and no complicated set up, so could just remove this test

Reply:
That sounds good to me